tfjs-node/src/kernels/Einsum.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
2023

Reply:
Good catch. Also, we no longer add 'All Rights Reserved' to new files since it may conflict with our OSS license.